pkg/config/kube/conversion.go

```diff
@@ -71,9 +72,18 @@ func ConvertProtocol(name string, proto coreV1.Protocol) config.Protocol {
 			name = name[:i]
 		}
 		protocol := config.ParseProtocol(name)
-		if protocol != config.ProtocolUDP && protocol != config.ProtocolUnsupported {
+
+		// TODO(crazyxy) skip for well known ports, e.e. prometheus
```

Comment:
do you want to start with a static map in pilot for well known ports? like 80, 443, 3306, 27012, etc? I am especially worried about server first protocols like mysql/mongo

Reply:
25 - SMTP
3306 - MySQL
8086 - InfluxDB
4222 - NATS
27017 - MongoDB

Reply:
Yeah. I think I can add a map for skipping well known ports. Except for well know ports, I think we should also add ports that istio uses.